### ISSUE TITLE
Litigation Support: Create new `SendCavcRemandProcessedLetterTask` type

### ISSUE THREAD

**Issue by araposo-tistatech** [department-of-veterans-affairs/caseflow#15076]
## User or job story
User story: As a Litigation Support, when I have completed the court remand kickoff form and have selected either a JMR or JMPR remand on the form, I need a "SendCavcRemandProcessedLetterTask" task to be created, so that I can take necessary action before sending the letter to an appellant. 

## Acceptance criteria
- [ ] This feature should be accessible to the following user groups: CAVC Litigation Support
- [ ] Consider better naming of SendCavcRemandProcessedLetterTask
- [ ] SendCavcRemandProcessedLetterTask is created on the new appeal stream as a child of the CAVC task
- [ ] SendCavcRemandProcessedLetterTask is assigned to the CAVC Litigation Support organization as unassigned
- [ ] The Actions drop-down has the following available for the Admin of the organization: 
- Assign to person
- [ ] Once assigned by the Admin of the organization, the Actions drop-down has the following available for the assigned user: 
- Add Admin Action
- Mark task complete
  - Indicates letter has been sent to veteran 

### Designs

[➡️ UI Designs and flow ⬅️
](https://www.figma.com/file/8noUqRbvCJzdpwkSNCb8lZ/CAVC-Remand-Designs?node-id=913%3A0)

**Comment by araposo-tistatech** [comment]
@geronimoramos please review and add the designs here. Let me know if I am not capturing this correctly. My understanding is Litigation Support will have a Send 90 Day Letter task, here they will create a POA task (that would block completing the task) as well as any other admin actions as necessary (these would not block completion of the task). The list I added is extensive and will most likely be limited given Litigation Supports response to our question of what admin actions they need available. 

I will create another task noting that upon completion of the Send 90 Day Letter task a Parent 90 Day Letter Response task will be created for the Litigation Support team on hold and a child 90 Day Letter Response task will be created or the mail team. 

Let me know if the above does not align with you. cc: @mbaenatan @hschallhorn

**Comment by mbaenatan** [comment]
This looks good and correct @araposo-tistatech. Still waiting on their guidance on admin actions, but I agree the only change would be to the list of admin actions based on their guidance.

**Comment by hschallhorn** [comment]
_[what is this chart?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | 
2 | ||
3 | ||||||
5 | |
8 | 
```

#### Why 2?
* Standard task creation, plenty of examples

#### Why 3?
* Basic addition - mimic existing classes

#### Why 5?
* Testing various scenarios

**Comment by yoomlam** [comment]
@geronimoramos or @araposo-tistatech 
1. Do we want the Lit Support admin to reassign this task to another user?
2. Do we want an assigned Lit Support non-admin to cancel this task?

**Comment by araposo-tistatech** [comment]
@yoomlam below are the answers, @geronimoramos please let me know if you disagree: 

1. Do we want the Lit Support admin or non-admin to reassign this task to another user? _Yes_

1. Do we want an assigned Lit Support non-admin to cancel this task? _Yes_

**Comment by araposo-tistatech** [comment]
@yoomlam Correcting #2 above: 

Do we want an assigned Lit Support non-admin to cancel this task? Since these are new users and this is the first task we should not give users the ability to cancel the task.
